src/Main/Main.module.scss

```diff
@@ -0,0 +1,6 @@
+@supports (position: sticky) {
+  .main {
+    padding-bottom: 4rem;
```

Comment:
Can this have a comment to explain it's a space for the back to top link to dock? Also, on CKS we use the `$back-to-top-height` variable. I'm not against just using 4rem but maybe a NDS spacing var would be better, and with a comment. Either way, without that you just have a magic number of 4rem which has significance